pd/src/sequential.rs

```diff
@@ -35,6 +35,8 @@ impl Sequencer {
         self.waiting = true;
         self.completion.set(rx);
 
+        // TODO: (@hdevalence) how do we propagate spans here?
```

Comment:
xref https://github.com/penumbra-zone/penumbra/pull/321/files#r777267153

Reply:
resolved in https://github.com/penumbra-zone/penumbra/pull/321/commits/8ccb8e13162b939e4d7d2fa524574b39cd977599